common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -686,15 +686,15 @@ def select_always_inline_discussion(self):
 
     def always_inline_discussion_selected(self):
         """
-        Returns the checked always_cohort_inline_discussions radio button.
+        Returns true if always_cohort_inline_discussions radio button is selected.
         """
-        return self.q(css=self._bounded_selector(".check-all-inline-discussions:checked"))
+        return len(self.q(css=self._bounded_selector(".check-all-inline-discussions:checked"))) > 0
```

Comment:
Note to reviewers: the way the previous method (and the one below) was used, the result of the assertion was always "True", regardless of actual select state (because [] is not None).